tests/conftest.py

```diff
@@ -81,26 +88,19 @@ def fixture_localhost(aiida_localhost):
     return localhost
 
 
-@pytest.fixture(scope="function")
-def janus_code(aiida_local_code_factory):
-    """
-    Fixture to get the janus code.
-
-    Parameters
-    ----------
-    aiida_local_code_factory : fixture
-        A fixture providing a factory for creating local codes.
-
-    Returns
-    -------
-    `Code`
-        The janus code instance.
-    """
+@pytest.fixture
+def janus_code(aiida_code_installed):
+    """Function-scoped fixture to get the janus Code."""
```

Comment:
Does this not work as a session scope? There's no obvious reason to need to reload the code each time